.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -31,6 +31,12 @@ A clear and concise description of what you expected to happen (or code).
 ```
 
 **Environment**
+<!--- Tryme: you can run `npx envinfo --preset babel` and paste the result below ``` -->
```

Comment:
I think we can simplify this a bit, with something like:

```suggestion
<!--- Tip: Instead of filling out the questions below, you can run `npx envinfo --preset babel` and paste the result below ``` -->
```

(And remove the subsequent two lines)